### PR TITLE
Adding sbt doctest

### PR DESCRIPTION
**Pull request by trentio**
I think this is something that we should be using along with actually publishing our scaladocs to someplace that we can access.

### CODE REVIEW

**Comment by adamdecaf** [comment]
:+1: Went ahead and made https://github.com/Banno/banno-sbt-plugin/pull/72 also

**Comment by zmccoy** [comment]
:+1: LGTM.  Anyone else have opinions?

File: src/main/scala/BannoSettings.scala [hunk 1]
```diff
@@ -21,6 +23,8 @@ object BannoSettings {
     Seq(javaOptions ++= Seq("-Djava.awt.headless=true", "-Xmx1024M", "-XX:MaxPermSize=512m")) ++
     GraphPlugin.graphSettings ++
     Revolver.settings ++
+    DoctestPlugin.doctestSettings ++
+    Seq(doctestTestFramework := DoctestTestFramework.Specs2, doctestWithDependencies := false) ++
```
Comment (rubbish) [inline src/main/scala/BannoSettings.scala]: Could we move these into another object/file like `Doctest`? Just trying to keep this tidy.

**Comment by rubbish** [comment]
:+1:

**Comment by rubbish** [comment]
@iliffwes @trentio We should do a talk on doctest/specs2?.

File: src/main/scala/BannoSettings.scala (hunk 1 above)
Comment (trentio) [inline src/main/scala/BannoSettings.scala]: Moving that now

**Comment by trentio** [comment]
@rubbish @iliffwes i could get something together for a couple weeks out.

File: src/main/scala/BannoSettings.scala (hunk 1 above)
Comment (trentio) [inline src/main/scala/BannoSettings.scala]: @rubbish done

**Comment by adamdecaf** [comment]
:+1:

**Pull request merged**